src/Inkluzitron/Contracts/IReactionHandler.cs

```diff
@@ -5,6 +5,6 @@ namespace Inkluzitron.Contracts
 {
     public interface IReactionHandler
     {
-        Task<bool> Handle(IUserMessage message, IEmote reaction, IUser user);
+        Task<bool> Handle(IUserMessage message, IEmote reaction, IUser user, IUser botUser);
```

Comment:
Zřejmě to bude na více místech, takto dám jako comment. 
Mohli bychom držet zavedený standard, který myslím, že má i samotnej .NET. A to, že asynchronní metody končí na Async. Takže tady by se to jmenovalo `HandleAsync`.